IntelliTect.TestTools.WindowsTestWrapper/IntelliTect.TestTools.WindowsTestWrapper/DesktopControls.cs

```diff
@@ -145,6 +145,17 @@ protected T FindWinControlByControlId<T>(string controlId, Func<UITestControl, T
             return found;
         }
 
+        protected T FindControlBySibling<T>(Func<UITestControl, T> controlType, UITestControl siblingControl ) where T : UITestControl
+        {
+            if (controlType == null)
+                throw new ArgumentNullException(nameof(controlType));
+            if (siblingControl == null)
+                throw new ArgumentNullException(nameof(siblingControl));
+            T found = controlType(siblingControl);
+            found.SearchConfigurations.Add(SearchConfiguration.NextSibling);
```

Comment:
This code is making the assumption that the `controlType` delegate never returns null. Is that a valid assumption?